src/Controller/AdController.php

```diff
@@ -29,7 +29,7 @@ public function index(AdRepository $adRepository): Response
     public function search(Request $request, AdRepository $adRepository): Response
     {
         $ads= [];
-        $ids = explode(",", $request->get('ids'));
+        $ids = array_filter(explode(",", $request->get('ids')));
         if ("" !== $ids[0]) {
```

Comment:
remove the `if`, with array_filter, it's not necessary because of the array_filter